appserver/batch/jbatch-repackaged/pom.xml

```diff
@@ -125,7 +125,7 @@
                         </Export-Package>
                         <Import-Package>
                             com.ibm.websphere.asynchbeans;resolution:=optional,
-                            javax.annotation;version="[1.1,2)";resolution:=optional,
+                            jakarta.annotation;version="[1.1,2)";resolution:=optional,
```

Comment:
This is a package name reference I don't think this should change as annotations has not changed package namespace.